flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesClusterDescriptorTest.java

```diff
@@ -86,7 +86,8 @@ public FlinkKubeClient fromConfiguration(
                                         server.createClient().inNamespace(NAMESPACE),
                                         Executors.newDirectExecutorService());
                             }
-                        });
+                        },
+                        config -> {});
```

Comment:
It is a valid point IMO, during the impl I was thinking about wiring a `KubernetesArtifactUploader.NO_OP` into `KubernetesClusterDescriptor` according to the `local-upload-enabled` config option, but I thought it would be better to encapsulate all this logic into the uploader so `KubernetesClusterDescriptor` won't manage any upload related logic actively, hence I did not added that `NO_OP` impl, cause in tests there are a lot of other in place no-op usage.